test/celeritas/optical/OpticalTestBase.hh

```diff
@@ -0,0 +1,65 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/optical/OpticalTestBase.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <memory>
+
+#include "celeritas/Quantities.hh"
+#include "celeritas/Types.hh"
+#include "celeritas/phys/ParticleData.hh"
+
+#include "DiagnosticRngEngine.hh"
+#include "celeritas_test.hh"
```

Comment:
I think you'll have to include `Test.hh` instead of this file, since `celeritas_test.hh` defines `main()`.